python-client/giskard/testing/tests/metamorphic.py

```diff
@@ -14,6 +16,7 @@
 from giskard.ml_worker.utils.logging import timer
 from giskard.models.base import BaseModel
 from giskard.models.utils import fix_seed
+from giskard.scanner.llm.utils import LLMImportError
```

Comment:
Maybe we should move `LLMImportError` outside of scanner now that we are integrating it more with the rest of the codebase. But not super important.